src/main.rs

```diff
@@ -191,6 +194,10 @@ struct Opt {
     #[clap(short = 'p', long = "print-package")]
     print_package: bool,
 
+    /// Print the package containing the executable
+    #[clap(short = 'p', long = "print-package-exit")]
```

Comment:
This short form is already used for the original print-package argument.

Reply:
Removed the short form

Reply:
To be honest, I don't really see the point of the other flag, why would you print the package only to then run it anyway?